config.toml.example

```diff
@@ -312,11 +312,11 @@
 
 # Whether or not debug assertions are enabled for the compiler and standard
 # library.
-#debug-assertions = false
+#debug-assertions = debug
```

Comment:
is this legal syntax if you uncomment it?

Reply:
It is invalid when uncommented. But I think it is valuable to indicate how default value is obtained, and similar approach is already used for other options.